apps/ledger-live-desktop/src/renderer/modals/SignMessage/Body.tsx

```diff
@@ -18,12 +18,15 @@ export type Data = {
   isACRE?: boolean;
 };
 
-type OwnProps = {
-  onClose?: () => void;
-  data: Data;
-};
-type Props = OwnProps;
-const steps: Array<St> = [
+const eip712Step = [
+  {
+    id: "sign",
+    label: "",
+    component: StepSign,
+  },
+];
+
+const messageSteps = [
```

Comment:
Small detail but you might want to keep the types

```suggestion
const eip712Step: Array<St> = [
  {
    id: "sign",
    label: "",
    component: StepSign,
  },
];

const messageSteps: Array<St> = [
```